File: datasets/IR-Plag/convertai-original-java-rest-python/case-03/plagiarized/L6/08/Kasus3L6.py
import math

def main():
    print("Enter weight in pounds: ", end="")
    weight = float(input())

    print("Enter feet: ", end="")
    feet = float(input())
    print("Enter inches: ", end="")
    inches = float(input())

    # Print Hasil perhitungan
    bmi = calculate_bmi(weight, (feet * 12) + inches)
    print_bmi_category(bmi)

def calculate_bmi(weight, height):
    temp = height * 0.0254
    return weight * 0.45359237 / (temp * temp)

def print_bmi_category(bmi):
    print(f"BMI is {bmi:.2f}")
    if bmi > 30:
        print("Obese")
    elif bmi > 25:
        print("Overweight")
    elif bmi > 18.5:
        print("Normal")
    else:
        print("Underweight")

if __name__ == "__main__":
    main()

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-03/plagiarized/L6/08/Kasus3L6.kt
package IR

import java.util.Scanner

fun main() {
    println("Enter weight in pounds: ")
    val inp = Scanner(System.`in`)
    val berat = inp.nextDouble()

    println("Enter feet: ")
    val kaki = inp.nextDouble()
    println("Enter inches: ")
    val inch = inp.nextDouble()

    // Print Hasil perhitungan
    BMIKategori(hitungBMI(berat, kaki * 12 + inch))
}

private fun hitungBMI(weight: Double, height: Double): Double {
    val temp = height * 0.0254
    return weight * 0.45359237 / (temp * temp)
}

private fun BMIKategori(BMI: Double) {
    println("BMI is $BMI")
    when {
        BMI > 30 -> println("Obese")
        BMI > 25 -> println("Overweight")
        BMI > 18.5 -> println("Normal")
        else -> println("Underweight")
    }
}

